src/pyVector3.py

```diff
@@ -0,0 +1,51 @@
+class pyvector3:
+    def __init__(self):
+        self._vec3 = [0, 0, 0]
+
+    def setFromQt(self, vecQt):
```

Comment:
(same applise for all `setFrom*` functions)
This approach clatters the user code, i.e. one have to repeteadly write the following sequence:
```
p = pyvector3()
p.setFromQt(some_qt_vector)
```
it would be much more convenient to use it like this:
```
p = pyvector3.setFromQt(some_qt_vector)
```

would it be a problem to implement it?

Reply:
done by removing pyvector3 class